source/titfront/src/components/Resizable.tsx

```
@@ -0,0 +1,78 @@
+        data-testid="resizable-content"
+        className="flex-grow size-full overflow-auto"
+        style={{ [dimension]: `${size}px` }}
+      >
+        {children}
+      </div>
+      <button
+        data-testid="resizable-resizer"
+        className="bg-gray-900"
+        style={{ cursor: cursor, [dimension]: "2px" }}
+        onMouseDown={handleMouseDown}
+      />
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)